packages/terra-clinical-onset-picker/tests/wdio/onset-spec.js

```diff
@@ -135,6 +135,7 @@ describe('Onset Picker', () => {
       browser.clearElement('[data-terra-clinical-onset-picker="age"] input');
       browser.setValue('[data-terra-clinical-onset-picker="age"] input', 10);
       browser.click('button[type="submit"]');
+      browser.pause(5000);
```

Comment:
Why are we adding this pause?

Reply:
The snapshots generated are highly inconsistent without the pause. The alert takes a second to load completely due to the animation.

Reply:
I thought there were some webdriverIO commands that would automatically wait until the element was on screen.  Can we use one of those instead before doing the click?  (http://webdriver.io/api/utility/waitForVisible.html or something similar)

Reply:
I tried that earlier. `waitForVisible` ( and other similar commands) do not recognize the completion of the animation.
Here's an example of the diff when pause is not used.

![image](https://user-images.githubusercontent.com/14154054/37614173-077e6fb4-2b78-11e8-9d04-20088e1476d6.png)